net-analyzer/ossec-hids/ossec-hids-3.1.0.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit user
+
+DESCRIPTION="Open Source Host-based Intrusion Detection System"
+HOMEPAGE="https://www.ossec.net/"
+SRC_URI="https://github.com/ossec/ossec-hids/archive/${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="GPL-2"
+SLOT="0"
+
+KEYWORDS="~amd64"
+
+IUSE="agent hybrid local mysql postgres server sqlite zlib"
+DEPEND="mysql? ( virtual/mysql )
+	sqlite? ( dev-db/sqlite:3 )
+	postgres? ( dev-db/postgresql:= )
+	zlib? ( sys-libs/zlib:= )"
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/${P}/src"
+
+MY_PREFIX="/var/ossec"
+
+my_options() {
+	local target="local"
```

Comment:
This really looks like a case for REQUIRED_USE w/ `^^`. Also, I think it'd make sense to have a default with `+` in IUSE.

Reply:
Database support is not a requirement, see [here](https://github.com/ossec/ossec-hids/blob/master/src/Makefile#L228).